examples/example_emscripten_wgpu/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# Example usage:
+#  cmake -B build -g "Ninja"
+
+cmake_minimum_required(VERSION 3.10.2)
+project(imgui_example_emscripten_wgpu C CXX)
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Debug CACHE STRING "" FORCE)
+endif()
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DVK_PROTOTYPES")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DVK_PROTOTYPES")
```

Comment:
Is it really needed? Compiled fine without this for me.

Reply:
Yeah these are not needed, was me being sloppy checking the cmake. (again copied from vulkan but this time I think vulkan needs those)